googletest/src/gtest.cc

```diff
@@ -3449,8 +3449,10 @@ class XmlUnitTestResultPrinter : public EmptyTestEventListener {
 XmlUnitTestResultPrinter::XmlUnitTestResultPrinter(const char* output_file)
     : output_file_(output_file) {
   if (output_file_.c_str() == NULL || output_file_.empty()) {
-    fprintf(stderr, "XML output file may not be null\n");
-    fflush(stderr);
+    {
```

Comment:
For each log to fatal FATAL we can trust GTEST_LOG_ to abort(), so we can skip the subsequent call to exit()

Reply:
So, it should just be GTEST_LOG_(FATAL) << "XML output file may not be null";
 and no need for { .. } scope